libraries/apollo-cli/src/main/kotlin/com/apollographql/apollo3/cli/DownloadSchemaCommand.kt

```diff
@@ -39,7 +40,8 @@ internal class DownloadSchemaCommand: CliktCommand() {
         registryUrl = registryUrl,
         schema = File(schema),
         headers = headersMap,
-        insecure =insecure,
+        insecure = insecure,
+        includeDeprecatedInputValuesAndArguments = includeDeprecatedInputValuesAndArguments,
```

Comment:
This should be either `includeDeprecatedInputValues` or `includeDeprecatedInputFieldsAndArguments`

Reply:
Oh yes good catch! I'll go with`includeDeprecatedInputFieldsAndArguments` (not a big fan of the long name but at least it's precise)